components/common/party/PartyModal.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div
+    class="party shadow-primary h-fit fixed !top-20 md:!top-24 sm:left-4 border-2 bg-background-color p-2 sm:!p-6 flex justify-between gap-2 sm:gap-6 max-w-[498px]">
+    <div class="flex-1 min-w-[91px] h-[116px] flex justify-center items-center">
+      <img :src="'/drop/party.svg'" />
+    </div>
```

Comment:
atm we don't get any kind of animation. it's pretty abrupt, check `CookieBanner.vue`

![CleanShot 2024-04-08 at 10 09 52@2x](https://github.com/kodadot/nft-gallery/assets/44554284/60b420f7-80b9-4576-b27b-abe616781042)

Reply:
I have added a <transition> wrapper but I can't see an animation on the cookiebanner as well :thinking: